manifests/agent.pp

```diff
@@ -216,7 +216,7 @@
       }else{
         fail('This ossec module has not been tested on your distribution')
       }
-    }'Debian', 'debian':{
+    }'Debian', 'debian', 'Ubuntu', 'ubuntu':{
```

Comment:
Maybe this changes that affect `agent.pp` and `manager.pp` should be implemented in another issue in order to have better tracking of it,

Reply:
Hi Jose!

Thank you for reviewing!

Here I created a new issue related to such change: https://github.com/wazuh/wazuh-puppet/issues/140

Kr,

Rshad